### PR TITLE
Updating reporting endpoints

### PR DESCRIPTION
**Pull request by nilakhan**
Taking forward Karan's change:

Rename daily and monthly user/printer usage report entities according to latest review with Graph
Rename periodStart and periodEnd arguments for retrieving archived jobs to startDateTime and endDateTime respectively.

### CODE REVIEW

**Review by braedenp-msft — APPROVED** [review]

File: api-reference/beta/api/printusagebyprinter-get.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-# Get printUsageSummaryByPrinter
+# Get printUsageByPrinter
 
 Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Retrieve a [printer](../resources/printer.md)'s usage summary for a particular time period. For descriptions of each of the endpoints, see [printUsageSummaryByPrinter](../resources/printUsageSummaryByPrinter.md).
+Retrieve a [printer](../resources/printer.md)'s usage summary for a particular time period. For descriptions of each of the endpoints, see [printUsageByPrinter](../resources/printUsageByPrinter.md).
```
Comment (isvargasmsft) [inline api-reference/beta/api/printusagebyprinter-get.md:17]: In general, we try to keep the first paragraph in a single sentence, so its easier to scan. Please include a blank line between the first and the second sentence:
- Retrieve a printer usage summary for a particular time period.
- (blank like here).
- For descriptions of each of the endpoints...

File: api-reference/beta/api/printusagebyprinter-get.md [hunk 2]
```diff
@@ -47,18 +47,18 @@ This method supports some of the OData query parameters to help customize the re
 ## Request body
 Do not supply a request body for this method.
 ## Response
-If successful, this method returns a `200 OK` response code and a [printUsageSummaryByPrinter](../resources/printusagesummarybyprinter.md) object in the response body.
+If successful, this method returns a `200 OK` response code and a [printUsageByPrinter](../resources/printUsageByPrinter.md) object in the response body.
 ## Example
 ##### Request
```
Comment (isvargasmsft) [inline api-reference/beta/api/printusagebyprinter-get.md:52]: Line 52: Request and Response headings in the Examples sections should be H3. The exception is when there are multiple examples.

File: api-reference/beta/api/printusagebyuser-get.md [hunk 3]
```diff
@@ -46,18 +46,18 @@ This method supports some of the OData query parameters to help customize the re
 ## Request body
 Do not supply a request body for this method.
 ## Response
-If successful, this method returns a `200 OK` response code and a [printUsageSummaryByUser](../resources/printusagesummarybyuser.md) object in the response body.
+If successful, this method returns a `200 OK` response code and a [printUsageByUser](../resources/printUsageByUser.md) object in the response body.
 ## Example
 ##### Request
```
Comment (isvargasmsft) [inline api-reference/beta/api/printusagebyuser-get.md:51]: Please use H3 headers here as well.

File: api-reference/beta/api/printusagebyuser-get.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
-# Get printUsageSummaryByUser
+# Get printUsageByUser
 
 Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Retrieve a user's usage summary for a particular time period. See the [printUsageSummaryByUser](../resources/printUsageSummaryByUser.md) documentation for descriptions of each of the endpoints.
+Retrieve a user's usage summary for a particular time period. See the [printUsageByUser](../resources/printUsageByUser.md) documentation for descriptions of each of the endpoints.
```
Comment (isvargasmsft) [inline api-reference/beta/api/printusagebyuser-get.md:16]: Please include a blank between the 1st and 2nd sentences.

File: api-reference/beta/api/reportroot-list-dailyprintusagebyprinter.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ GET /print/reports/dailyPrintUsageSummariesByPrinter
 ## Request body
 Do not supply a request body for this method.
 ## Response
-If successful, this method returns a `200 OK` response code and a collection of [printUsageSummaryByPrinter](../resources/printusagesummarybyprinter.md) objects in the response body.
+If successful, this method returns a `200 OK` response code and a collection of [printUsageByPrinter](../resources/printUsageByPrinter.md) objects in the response body.
 ## Example
 ##### Request
```
Comment (isvargasmsft) [inline api-reference/beta/api/reportroot-list-dailyprintusagebyprinter.md:46]: Please use H3 headers for the Request and Response sections in the examples.

File: api-reference/beta/api/reportroot-list-dailyprintusagebyuser.md [hunk 6]
```diff
@@ -44,7 +44,7 @@ This method supports some of the OData query parameters to help customize the re
 ## Request body
 Do not supply a request body for this method.
 ## Response
-If successful, this method returns a `200 OK` response code and a collection of [printUsageSummaryByUser](../resources/printusagesummarybyuser.md) objects in the response body.
+If successful, this method returns a `200 OK` response code and a collection of [printUsageByUser](../resources/printUsageByUser.md) objects in the response body.
 ## Example
 ##### Request
```
Comment (isvargasmsft) [inline api-reference/beta/api/reportroot-list-dailyprintusagebyuser.md:49]: H3 as well, please.

File: changelog/Microsoft.PrintService.json [hunk 7]
```diff
@@ -1,5 +1,167 @@
...
+        {
+          "Id": "dc7d0b6f-64f8-11eb-8de9-f7f8b8b7f046",
+          "ApiChange": "Method",
+          "ChangedApiName": "getPrinterArchivedPrintJobs",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **getPrinterArchivedPrintJobs** method from the [reportRoot](https://docs.microsoft.com/en-us/graph/api/resources/reportRoot?view=graph-rest-beta) resource",
+          "Target": "reportRoot"
+        }
+      ],
+      "Id": "dc4d5c05-64f8-11eb-8de9-f7f8b8b7f046",
```
Comment (isvargasmsft) [inline changelog/Microsoft.PrintService.json:158]: All elements in a single changelist should share the same GUID. Would you be so kind to use the same GUID across your Changelist, please?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
@nilakhan thank you very much for the careful update to this docs. Great work!
Just a couple of suggestions:
- Please use H3 headers for the Request and Response sections inside examples. 
- Please use a single GUID for all elements inside a ChangeList in the changelog. 

Thanks, 

Isaac Vargas

File: api-reference/beta/api/printusagebyprinter-get.md [hunk 8]
```diff
@@ -47,18 +49,18 @@ This method supports some of the OData query parameters to help customize the re
...
 # [HTTP](#tab/http)
 <!-- {
   "blockType": "request",
-  "name": "get_printUsageSummaryByPrinter"
+  "name": "get_printUsageByPrinter"
 }-->
-```msgraph-interactive
-GET https://graph.microsoft.com/beta/print/reports/dailyPrintUsageSummariesByPrinter/{id}
+```http
+GET https://graph.microsoft.com/beta/print/reports/dailyPrintUsageByPrinter/{id}
```
Comment (isvargasmsft) [inline api-reference/beta/api/printusagebyprinter-get.md:63]: Would you be so kind to add a real looking ID to the examples? We try to use placeholders in the HTTP Request section above, but real looking data in the examples. 

It doesn't have to be a real ID, but one that looks like a real one. Once you do that, please include the SampleKeys attribute to the API Doctor block in lines 58 to 61. More details [here](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103189/Set-up-for-API-Doctor):

> sampleKeys: Specify this for example HTTP requests that include one or more ID values indexing into collections in the request query. This property is a collection of strings, each string representing an ID value in the example request. For more information about specifying real ID values in examples, see the section ID values in example requests.

File: api-reference/beta/api/printusagebyuser-get.md [hunk 9]
```diff
@@ -46,18 +48,18 @@ This method supports some of the OData query parameters to help customize the re
...
 # [HTTP](#tab/http)
 <!-- {
   "blockType": "request",
-  "name": "get_printUsageSummaryByUser"
+  "name": "get_printUsageByUser"
 }-->
-```msgraph-interactive
-GET https://graph.microsoft.com/beta/print/reports/dailyPrintUsageSummariesByUser/{id}
+```http
+GET https://graph.microsoft.com/beta/print/reports/dailyPrintUsageByUser/{id}
```
Comment (isvargasmsft) [inline api-reference/beta/api/printusagebyuser-get.md:62]: Same observation here. Sorry for not catching it in the first review. Please add real looking IDs to the examples and include sampleKeys attribute in the API Doctor block.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Just a couple of additional changes. Please include real looking ID in the examples. 

Thank you very much!

File: api-reference/beta/api/printusagebyprinter-get.md (hunk 8 above)
Comment (nilakhan) [reply api-reference/beta/api/printusagebyprinter-get.md]: Added

**Comment by Lauragra** [comment]
@isvargasmsft , if this PR looks good on your end can you please approve? Thanks.

**Review by isvargasmsft — APPROVED** [review]
Thank you very much! Great work!

**Pull request merged**